rasa/cli/data.py

```diff
@@ -643,15 +643,23 @@ def _backup(path: Path) -> None:
 
 
 def _print_success_message(new_rules: List["StoryStep"], output_file: Path) -> None:
-    if len(new_rules) > 1:
-        suffix = "rule"
-        verb = "was"
-    else:
+    if len(new_rules) > 1 or len(new_rules) == 0:
         suffix = "rules"
         verb = "were"
+    else:
+        suffix = "rule"
```

Comment:
I know this was already like this, but `suffix` seems like the wrong name

Reply:
Changed it to `rules_text`.